katalog/mimir/config/mimir.yaml

```diff
@@ -58,24 +58,22 @@ limits:
   max_cache_freshness: 10m
   max_query_parallelism: 240
   max_total_query_length: 12000h
-  compactor_blocks_retention_period: 720h # 30 days retention by default
```

Comment:
Why was this removed?

Reply:
Taken from upstream, we should keep the property from now on